hugo/layouts/why-sourced/single.html

```diff
@@ -0,0 +1,170 @@
+                <div class="circle purple-text small-text mb-5">2020</div>
+                <h3 class="text-dark">Observability</h3>
+                <p class="text-dark">Shared data, common processes and end to end visibility allowing business and IT leaders to collaborate effectively</p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="effective" class="effective py-5">
+    <div class="container text-dark">
+        <header class="text-center mb-5">
+            <h2>With no data, enterprises can not level up their Engineering Effectiveness</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-6">
+                <p>Lack of visibility comes from:</p>
+                <ul class="small-list">
+                    <li>Volume, variety, intricacy and versioning of source code</li>
+                    <li>No easy way to retrieve, parse and query source code history at scale</li>
+                    <li>Lack of proper documentation and/or expertise with legacy systems</li>
+                    <li>Source code, software development and business data are spread across many silos</li>
+                    <li>Shadow IT due to developer frustration with legacy systems & slow processes</li>
+                </ul>
+            </div>
+            <div class="col-sm-6">
+                <img class="img-fluid" src="img/visibility.jpg" alt="">
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="success">
+    <div class="container">
+        <header class="text-center text-dark my-5">
+            <h2>source{d} increases the success rates and the benefits  of your IT modernization projects</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-6 mb-5">
+                <div class="card text-white green">
+                    <div class="card-body p-4 p-lg-5">
+                        <p class="card-intro">Very rewarding</p>
+                        <h3 class="card-title">$1.4 billion<br> <span class="h4">anual incremental revenues</span></h3>
+                        <p class="card-text">Potential impact of the right transformation team over 9 months</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-6">
+                <ul class="small-list">
+                    <li>Reduces infraestructure and/or development costs</li>
+                    <li>Faster time to market & issue resolution</li>
+                    <li>Higher developer productivity & morale</li>
+                    <li>Shifting IT budget from maintenance to innovation, from CapEx to OpEx</li>
+                    <li>Tech stack is more appealing to engineering talents</li>
+                </ul>
+            </div>
+            <div class="col-sm-12 text-center py-5">
+                <a href="#" class="btn btn-purple text-white mr-4">Download source{d}</a>
+                <a href="#" class="btn btn-highlight">Request a Demo</a>
```

Comment:
These two buttons have no proper margin in mobile
![image](https://user-images.githubusercontent.com/2437584/58349094-a87a6880-7e62-11e9-80e0-4f3fb20ab05f.png)

Reply:
Fixed